### ISSUE TITLE
[Sf 3.3] Error on edit missing message in Profiler

### ISSUE THREAD

**Issue by welcoMattic** [php-translation/symfony-bundle#95]
Hi, I'm using the bundle in Sf 3.3-BETA. No problem during installation. But after apply a simple config:

```yaml
translation:
    locales: ['fr', 'en']
    symfony_profiler:
        enabled: true
    configs:
        app:
            dirs: ["%kernel.root_dir%/Resources/views", "%kernel.root_dir%/../src"]
            output_dir: "%kernel.root_dir%/Resources/translations"
            excluded_names: ["*TestCase.php", "*Test.php"]
            excluded_dirs: [cache, data, logs]
```

When I want to `edit` a missing message in SfProfiler, I catch this error in `Network` tab in DevTools (url: `http://myapp.dev/app_dev.php/19ed84/translation/edit?message_id=frmessagessearch.form.origin.label`).

> Type error: Argument 1 passed to Translation\Bundle\Model\SfProfilerMessage::create() must be of the type array, object given, called in /var/www/vendor/php-translation/symfony-bundle/Controller/SymfonyProfilerController.php on line 160

If you need more information, let me know.

**Comment by Nyholm** [comment]
Thank you for this issue and PR. I will try to review it tonight.

**Comment by welcoMattic** [comment]
Do you plan to release a `0.3.2` version with this fix? Also with a `0.2.2` of https://github.com/FriendsOfApi/localise.biz ?

**Comment by Nyholm** [comment]
Sure

**Comment by Nyholm** [comment]
I've released new 0.2.2 of Localise. 

We need to update the changelog here. Feel free to do it if you want.  I can do it but later today. 
Could this issue be closed?

**Comment by welcoMattic** [comment]
Nice, thank you. I'll write the changelog later today.

Yes, we can close this issue
